quarkus/config-api/src/main/java/org/keycloak/config/Option.java

```diff
@@ -24,7 +25,7 @@ public Option(Class<T> type, String key, OptionCategory category, boolean hidden
         this.buildTime = buildTime;
         this.description = getDescriptionByCategorySupportLevel(description, category);
         this.defaultValue = defaultValue;
-        this.expectedValues = expectedValues;
+        this.expectedValues = expectedValues.get();
```

Comment:
I'd choose only one approach, how to specify the expected values. AFAIK, the Supplier for the expected values was intended to use for lazy evaluation of the values based on environment/configuration; dynamic values. In this case, it doesn't have any effect as it's resolved in the constructor.

AFAIK, for all options we have in Keycloak, we use only static not changeable values. I think we can remove the Supplier as there's no need for it now.

@shawkins WDYT?

Reply:
> the Supplier for the expected values was intended to use for lazy evaluation

But the Supplier was typically backed by an innner class that held the list of items. The only case where that was not true was with the list of features.

> AFAIK, for all options we have in Keycloak, we use only static not changeable values. I think we can remove the Supplier as there's no need for it now.

I'm fine with that too, was just trying to minimize changes.